mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -112,6 +126,8 @@ CLASS net/minecraft/class_1959 net/minecraft/world/biome/Biome
 		METHOD method_8743 scale (F)Lnet/minecraft/class_1959$class_1960;
 			ARG 1 scale
 		METHOD method_8745 parent (Ljava/lang/String;)Lnet/minecraft/class_1959$class_1960;
+			COMMENT Sets the biome that this will replace as a modified version of the biome.
+			COMMENT @param parent the string identifier of the biome to be replaced
```

Comment:
Please comment on the parameter instead of adding a param tag on the method.

Reply:
wdym?

Reply:
like this? (see new change)

Reply:
oh wait

Reply:
yeah I see what you mean lol

Reply:
done